doc/tokens/tokentypes/yubikey.rst

```diff
@@ -3,22 +3,33 @@
 Yubikey
 -------
 
-.. index:: Yubikey, Yubico AES mode
+.. index:: Yubikey, Yubico AES mode, Yubikey OATH-HOTP mode
 
-The Yubikey is initialized with privacyIDEA and works in Yubicos own AES mode.
-It outputs a 44 character OTP value, consisting of a 12 character prefix and
+The Yubikey can be used with privacyIDEA in Yubico's own AES mode (*Yubico OTP*)
+or in the OATH-HOTP mode which is for privacyIDEA a HOTP Token.
```

Comment:
I would rephrase this and talk of a token type rather then a token:

    or in the OATH-HOTP mode which is an HOTP token type for privacyIDEA.